providers/snowflake/schema_grant.go

```diff
@@ -82,7 +82,7 @@ func (g SchemaGrantGenerator) createResources(schemaGrantList []schemaGrant) ([]
 	for id, grant := range groupedResources {
 		resources = append(resources, terraform_utils.NewResource(
 			id,
-			fmt.Sprintf("%s_%s", grant.Name, grant.Privilege),
+			fmt.Sprintf("%s_%s", strings.Replace(grant.Name, ".", "__", 1), grant.Privilege),
```

Comment:
oh, I see what is going on.
it might be less confusing if you do like before, split on the . and then have
`fmt.Sprintf("%s__%s__%s", db, schema, priv)`

Reply:
up on line 62 https://github.com/chanzuckerberg/terraformer/pull/18/files#diff-57d0435478d58fae82b22afc46673ef2R62

Reply:
alternatively, augment the TFgrant struct with DBNAme and Schema name